t/file.t

```diff
@@ -63,3 +65,14 @@ for my $t (@tests) {
    print "ok $testno\n";
    $testno++;
 }
+
+my $file = ' hello world ';
+my $u = URI::file->new_abs( $file );
+my @dirs = File::Spec->splitdir( $u->file( 'Unix' ) );
+print( "not " ) if( $dirs[-1] ne $file );
+print( "ok $num\n" );
```

Comment:
This seems to assume that this test will always be the last in the script.

It seems like it would be better if implemented as its own free-standing test script?